src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1536,7 +1536,12 @@ The file `PROACTIVE_HOME/config/security.java.policy-server` allows to configure
 
 * Deploy ProActive Nodes
 * Execute jobs
-* Pause the Scheduler
+* Perform administrator actions from the scheduler portal like:
+** Start the scheduler: Jobs can be submitted. Getting the jobs results is possible.
+** Stop the scheduler: Jobs cannot be submitted anymore. This terminates every submitted jobs. Getting the jobs results is possible.
```

Comment:
I think it terminates only running or stalled jobs, pending jobs are not executed.

Reply:
Stop the scheduler: Jobs cannot be submitted anymore. Already running jobs run to completion, but not pending jobs. Getting the jobs results is possible. The scheduler can be started or killed.

Reply:
(tested)